test/Common/ServerFixture.cs

```diff
@@ -44,32 +42,22 @@ private void StartServer()
                 .UseContentRoot(Directory.GetCurrentDirectory())
                 .Build();
 
-            var t = Task.Run(() => _host.Start());
             _logger.LogInformation("Starting test server...");
-            _lifetime = _host.Services.GetRequiredService<IApplicationLifetime>();
-            if (!_lifetime.ApplicationStarted.WaitHandle.WaitOne(TimeSpan.FromSeconds(5)))
+            using (var cts = new CancellationTokenSource())
             {
-                // t probably faulted
-                if (t.IsFaulted)
-                {
-                    throw t.Exception.InnerException;
-                }
-                throw new TimeoutException("Timed out waiting for application to start.");
+                cts.CancelAfter(TimeSpan.FromSeconds(30));
```

Comment:
Why do we need to time this out ever?

Reply:
We want to be safe, don't want it to hang for any reason.

Reply:
Lets remove this.